src/codeu/chat/common/SentimentScore.java

```diff
@@ -46,16 +45,16 @@
   public static final Serializer<SentimentScore> SERIALIZER = new Serializer<SentimentScore>() {
     @Override
     public void write(OutputStream out, SentimentScore value) throws IOException {
-      Serializers.INTEGER.write(out, value.score);
-      Serializers.INTEGER.write(out, value.numScores);
+      Serializers.DOUBLE.write(out, value.score);
+      Serializers.DOUBLE.write(out, value.weighting);
```

Comment:
If these scores are only DOUBLEs in the sense that they're to 2 decimal places, it's usually best to use an integer multiplier instead so you don't have to deal with floating point math.